webrtc.html

```diff
@@ -7149,6 +7155,36 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           <var>channel</var>.</p>
         </li>
       </ol>
+      <p>In some cases, the user agent may be unable to create an
+      <code><a>RTCDataChannel</a></code>'s <a>underlying data transport</a>.
+      For example, the data channel's <code><a
+      data-for="RTCDataChannel">id</a></code> may be outside the range
+      negotiated by the [[!RTCWEB-DATA]] implementations, or in the case where
+      the user agent generates an id according to [[!RTCWEB-DATA-PROTOCOL]], no
+      more IDs may be available. When the user agent determines that an
+      <code><a>RTCDataChannel</a></code>'s <a>underlying data transport</a>
```

Comment:
> What's an "underlying data transport"?

My assumption was that it refers to a DataChannel as defined by RTCWEB-DATA.

> Why not just say that an ID could not be allocated?

I wanted to be as specific as possible (by providing examples), but also use terminology consistent with the rest of this section. I think "the underlying data transport cannot be created" accomplishes that.